shift_detector/precalculations/count_vectorizer.py

```diff
@@ -0,0 +1,87 @@
+import pandas as pd
+from shift_detector.precalculations.precalculation import Precalculation
+from nltk.corpus import stopwords as nltk_stopwords
+from sklearn.feature_extraction.text import CountVectorizer as CountVectorizer_sklearn
+from shift_detector.utils.column_management import ColumnType
+
+
+class CountVectorizer(Precalculation):
+
+    def __init__(self, cols, stop_words='english', max_features=None):
+        # potentially make min_df and max_df available
+        self.stopwords = None
+        self.max_features = None
+        self.cols = None
+        if isinstance(stop_words, str):
+            if stop_words in nltk_stopwords.fileids():
+                self.stop_words = nltk_stopwords.words(stop_words)
+            else:
+                raise Exception('The language you entered is not available')
```

Comment:
Do not use generic exceptions. If none of the default errors fit, create your own